### DISCUSSION TITLE
WDS 5 migration help

### DISCUSSION THREAD

**Discussion by RDIL** [webpack/webpack#18868] [Q&A]
Hi all,

I'm working on a webpack 5 migration, and the dev-server part has been a particularly gnarly migration, as we wrap infrastructure around it in some weird ways. Particularly, the `public`/`host` changes have been quite weird. This is the relevant section of the config:

```js
devServer: {
    host: '0.0.0.0',
    hot: false,
    lazy: false,
    inline: false,
    port: 3002,
    public: process.env.WEBPACK_DEV_SERVER_HOST || 'webpack.localhost',
    publicPath: '/compiled/',
    quiet: true,
    headers: {
      'Access-Control-Allow-Origin': '*',
    },
},
```

I tried to switch this over to the new `sockHost` property, but I started getting an invalid Host header error when trying to fetch the assets in my browser. Can someone provide assistance with this? Thank you!

**Comment by alexander-akait** [comment]
Just use `host`, you don't need `public` at all

**Reply by RDIL** [reply]
I'll try that, thanks!
